tests/e2e/machine_pool_test.go

```diff
@@ -614,6 +614,41 @@ var _ = Describe("TF Test", func() {
 
 			})
 		})
+		Context("Author:amalykhi-Medium-OCP-65645 @ocp-65645 @amalykhi", func() {
+			It("Author:amalykhi-Medium-OCP-65645 MP reconciliation basic flow", ci.Day2, ci.Medium, ci.FeatureMachinepool, func() {
+				By("Create additional machinepool with taints")
+				replicas := 3
+				machineType := "r5.xlarge"
+				mpName := "ocp-65645"
+				taint0 := map[string]string{"key": "k1", "value": "val", "schedule_type": con.NoExecute}
+				taints := []map[string]string{taint0}
+				MachinePoolArgs := &exe.MachinePoolArgs{
+					Cluster:     clusterID,
+					Replicas:    &replicas,
+					MachineType: machineType,
+					Name:        mpName,
+					Taints:      taints,
+				}
+
+				_, err := mpService.Apply(MachinePoolArgs, false)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Verify the parameters of the created machinepool")
+				_, err = cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, mpName)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Delete machinepool by OCM API")
+				cms.DeleteMachinePool(ci.RHCSConnection, clusterID, mpName)
+				_, err = cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, mpName)
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).Should(ContainSubstring("Machine pool with id '%s' not found", mpName))
+
+				By("ReApply the machinepool manifest")
+				_, err = mpService.Apply(MachinePoolArgs, false)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
maybe worth adding another `By("Verify the parameters of the created machinepool")` ?
just to check the machine pool creation?